school/lsoa/region_stats.Rmd

```diff
@@ -52,7 +52,7 @@ There were `r prettyNum(sum(z$all), big.mark = ",", scientific = F)` school chil
 
 There were `r prettyNum(sum(d$all), big.mark = ",", scientific = F)` school children attending a school in `r get_pretty_region_name(region)` recorded in the 2011 NSC. All of these children are included in the **destinations** data that are visualised on the map interface and available for download.  The interface pop-ups do not, however, present exact numbers for counts based on 1 to 5 children.
 
-Finally, the **Route Network (LSOA, clickable)** is built from all origin-destination fast routes within the region that contain at least `r region_build_param$minflow_rnet` children and have a fast route distance <5km (for primary school children)/<10km (for secondary school children). The pop-ups do not present exact numbers for segments containing 1 to 2 cyclists.
+Finally, the **Route Network (LSOA, clickable)** and the **Route Network (LSOA, image)** are  built from all origin-destination fast routes within the region. In the online version of the 'clickable' route network, we only present route segments that contain at least `r region_build_param$mingodutch_rnet` cyclists in the Go Dutch scenario.  All route segments are included in the versions that can be downloaded. The pop-ups and downloads do not present exact numbers for segments containing 1 to 2 cyclists.
```

Comment:
This is correct, except 

> All route segments are included in the versions that can be downloaded

Sould be replaced with

> All route segments for which the Go Dutch values are greater than 10 are included in the versions that can be downloaded

Reply:
ok, changing it

Reply:
Thanks.